buildSrc/build.gradle

```diff
@@ -13,6 +13,7 @@ apply from: rootProject.file('../dependencies.gradle')
 
 repositories {
   mavenCentral()
```

Comment:
[As far as I understood](http://stackoverflow.com/questions/24852219/android-buildscript-repositories-jcenter-vs-mavencentral) Jcenter is a superset of mavenCentral, so you should not need this one (same in the above gradle file).